Documentation/install/guides/kops.rst

```diff
@@ -152,6 +152,7 @@ The default Cilium version deployed by ``kops`` is old. Upgrade the Cilium Daemo
 
 .. code:: bash
 
+        kubectl delete crd ciliumendpoints.cilium.io # this ensures older CEP objects do not persist
```

Comment:
Do you know if we are tracking updating the version of Cilium deployed with kops upstream anywhere?